### PR TITLE
fix: ui updates for market research pages

### LINKED ISSUE #4292: Address frontend UI issues on marketing research pages
- [x] update color of gradient button
- [x] Double-check spacing in features
- [x] Max width paragraph features
- [x] What’s included text should be a smaller font size
- [x] Align purple strip of deep dives image with container
- [x] Readable line length for lead text in hero
- [x] Deep dive lead left aligned
- [x] Article sidebar component padding desktop
- [x] Last p element should not have bottom margin
- [x] Link color changes to blue 50
- [x] h1, h2, h3, h4 -> metropolis 600
- [x] Double-check spacing between headings
- [x] Survey panels should use visually-hidden
- [x] See if I can address the font issue with the main menu
- [x] Update lorem ipsum on browser support toolkit article

### LINKED ISSUE #4294: Pass 2: Address frontend UI issues on marketing research pages
Changes based on the following doc:
https://docs.google.com/document/d/1djYq3gXgNKRfwLFcju6a-RiOpmYVF5f0oEjIrXtk1zY/edit?ts=60fad14c#

#### standard desktop [1366 x 768]

- [x] Font sizes are larger for “MDN Plus” and title
- [x] Margin is larger

#### phone

- [x] Hero section is smaller
- [x] More MDN. Your MDN. Second line is completely on second line for mobile
- [x] Font is smaller on mobile
- [x] Margin on top of hero is smaller, margin on sides of hero is smaller (5% padding)

#### standard desktop [1366 x 768]

- [x] Background is different color
- [x] Margin on sides is larger
- [x] Text should fit onto 5 lines on both sides
- [x] Bold is bolder (600)

#### phone

- [x] Fonts are smaller for header and text

#### standard desktop [1366 x 768]

- [x] Margins are larger
- [x] Boost your learning is larger
- [x] Spacing between Boost your learning is larger
- [x] Font is smaller for body text
- [x] Last item on bulleted list is indented

#### phone

- [x] Even spacing between subheader, header, and text
- [x] Text is smaller
- [x] Bullets should be indented if text goes over more than one line in list
- [x] Illustration is larger (should fit entire container with 5% padding)

#### standard desktop [1366 x 768]

- [x] Background is darker
- [x] Card is 1fr and content is 2fr
- [x] Margin is larger
- [x] Card: Featured Deep Dive is bolder
- [x] Card: Modern CSS in the real world is larger
- [x] Card is wider
- [ ] Article text is larger
- [x] Button is thinner
- [ ] Code samples are rounded completely
- [x] Deep dive card is sticky

#### phone

- [ ] More padding between sections
- [x] This three part series is smaller

#### standard desktop [1366 x 768]

- [x] Margins are larger on sides
- [x] More space between illustrations and text
- [x] Top bar in illustration is aligned to section background
- [x] Bottom of illustration is same distance away from section background as top

#### phone

- [x] Bottom of illustration is same distance away from section background as top

#### phone

- [x] More space between feature sections

### PR DESCRIPTION
**Pull request by schalkneethling**
The first iteration of UI updates to market research pages.

fix #4292

### CODE REVIEW

File: client/src/plus/deep-dives/sample-articles.tsx [hunk 1]
```diff
@@ -395,8 +395,8 @@ export function YourBrowserSupportToolkit() {
         <h2>Modern CSS in the real world : Part two</h2>
       </header>
       <p className="article-lead">
-        Viverra neque eget tellus, gravida pellentesque non. Purus, venenatis
-        sit ac nam nec. At vitae facilisi in mi amet aliquam.
+        In this article discover the resources available, to help you develop a
+        site that will perform well across browsers and devices.
```
Comment (peterbe) [inline client/src/plus/deep-dives/sample-articles.tsx:399]: This is incorrect now. Merge in the latest `main` and resolve the conflict.

Comment (schalkneethling) [reply client/src/plus/deep-dives/sample-articles.tsx:399]: Done.

**Review by peterbe — COMMENTED** [review]
Was this meant to fix the main menu? 
Screen Shot 2021-07-23 at 10 35 27 AM

File: client/src/plus/deep-dives/sample-articles.tsx (hunk 1 above)
Comment (peterbe) [reply client/src/plus/deep-dives/sample-articles.tsx:399]: I don't get why but she wants the sentences to now end with a period. 
See https://github.com/mdn/yari/pull/4291

**Comment by schalkneethling** [comment]
> Was this meant to fix the main menu?

Just pushed the additional change. So used to looking at the new design, I missed that they are currently bold.

File: client/src/plus/deep-dives/sample-articles.tsx (hunk 1 above)
Comment (schalkneethling) [reply client/src/plus/deep-dives/sample-articles.tsx:399]: It does have a period though? Or are you saying there should not be a period?

Comment (peterbe) [reply client/src/plus/deep-dives/sample-articles.tsx:399]: > It does have a period though? Or are you saying there should not be a period?

Not have a period. Don't ask me why. 
See conversation in https://github.com/mdn/yari/pull/4291

**Comment by peterbe** [comment]
It's still different. 
Screen Shot 2021-07-23 at 10 50 19 AM

File: client/src/plus/deep-dives/article.tsx [hunk 2]
```diff
@@ -95,7 +95,13 @@ export default function Article({ slug }: { slug: string }) {
       </div>
       <Feature />
       <Survey slug={slug} />
-      <div className="deep-dive-article-footer">
+      <div
+        className={
+          previousArticle
+            ? "deep-dive-article-footer previous"
+            : "deep-dive-article-footer"
+        }
```
Comment (peterbe) [inline client/src/plus/deep-dives/article.tsx:103]:
```jsx
className={`deep-dive-article-footer ${previousArticle ? "previous": ""}`}
```

**Comment by schalkneethling** [comment]
> It's still different.

I reckon that is fine for now.

**Review by peterbe — APPROVED** [review]
It's a bit hard to review this. I don't know what I'm supposed to review. 
The PR description links to another PR. 

I'm aware that I don't speak SCSS fluent enough to review what you've typed in, but it would have been nice to spell out what this PR does so I can see if the code matches those expectations. Even if it just mentioned what was wrong/bad before. 

At least it doesn't crash and I can read the deep dives.

**Comment by peterbe** [comment]
> fix #4293

Probably a typo.

**Pull request merged**